app-api/src/main/java/org/openmrs/eip/app/management/entity/SenderSyncMessage.java

```diff
@@ -0,0 +1,104 @@
+package org.openmrs.eip.app.management.entity;
+
+import javax.persistence.Column;
+import javax.persistence.Entity;
+import javax.persistence.Table;
+import javax.validation.constraints.NotNull;
+
+@Entity
+@Table(name = "sender_sync_message")
+public class SenderSyncMessage extends AbstractEntity {
+	
+	private static final long serialVersionUID = 1L;
+	
+	@NotNull
+	@Column(name = "table_name", length = 100, nullable = false, updatable = false)
+	private String tableName;
+	
+	@NotNull
+	@Column(length = 255, nullable = false, updatable = false)
+	private String identifier;
+	
+	@NotNull
+	@Column(length = 1, nullable = false, updatable = false)
+	private String operation;
+	
+	@NotNull
+	@Column(name = "message_uuid", length = 38, nullable = false, unique = true, updatable = false)
+	private String messageUuid;
+	
+	@Column(name = "request_uuid", length = 38, unique = true, updatable = false)
+	private String requestUuid;
+	
+	@NotNull
+	@Column(length = 1, nullable = false)
```

Comment:
You don't need to specify length for a boolean column

Reply:
Right, fixed.